### PR TITLE
Display Resque task args in progress

### PR DESCRIPTION
**Pull request by evantahler**
Updates the resque workers page to:
* have linkable anchors for navigation
* hide the long list of queues until expanded
* show the arguments of the task in progress

Screen Shot 2021-11-16 at 4 40 01 PM

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: ui/ui-components/pages/resque/overview.tsx [hunk 1]
```diff
@@ -214,11 +214,25 @@ export default function ResqueOverview(props) {
...
                         <td>
-                          <span
-                            className={worker.delta > 0 ? "text-success" : ""}
+                          <Link
+                            href={
+                              "/resque/workers#" +
+                              `${worker.host}:${worker.id}`.replace(
+                                /[\W_-]/g,
+                                "-"
+                              )
```
Comment (evantahler) [inline ui/ui-components/pages/resque/overview.tsx:223]: The worker names have `:` and `.` in them - we need to clean them up to be valid HTML anchors

File: ui/ui-components/pages/resque/workers.tsx [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
+type DisplayedWorker = {
+  id: string;
+  name: string;
+  host: string;
+  queues: string[];
+  status: "working" | "pending";
+  job: string;
+  args: any[];
+  delta: number;
+};
```
Comment (evantahler) [inline ui/ui-components/pages/resque/workers.tsx:21]: Now with a real type!

File: ui/ui-components/pages/resque/overview.tsx (hunk 1 above)
Comment (teallarson) [reply ui/ui-components/pages/resque/overview.tsx:223]: 👍  I don't think we need the "-" to be replaced if the replacement is "-" though.

File: .circleci/config.yml [hunk 3]
```diff
@@ -1533,6 +1533,8 @@ jobs:
           name: test
           command: cd plugins/@grouparoo/calculated-property && ./node_modules/.bin/jest  --ci --passWithNoTests --runInBand
   test-plugin-clickhouse:
+    environment:
+      GROUPAROO_TELEMETRY_ENABLED: "false"
```
Comment (teallarson) [inline .circleci/config.yml:1537]: 🎉

**Review by teallarson — APPROVED** [review]
This will be super helpful for debugging!  Nice.

**Pull request merged**